client/js/table-view.js

```diff
@@ -74,6 +77,20 @@ class TableView {
 		this.sheetBodyEl.appendChild(fragment);
 	}
 
+	renderTableFooter() {
+		const tfootFragment = document.createDocumentFragment();
+			// create data cells for row based on number of columns
```

Comment:
Please use spaces not tabs.  Set your editor accordingly.  Confer with Kay if you aren't sure how to do this.